### DISCUSSION TITLE
minio insert_log目录占用过多空间？

### DISCUSSION THREAD

**Discussion by luokuncool** [milvus-io/milvus#21377] [Q&A and General discussion] [answered]
![image](https://user-images.githubusercontent.com/6022635/209461667-310a2c97-51b5-46b0-91f6-d9f0a3957d5e.png)
minio，里面的这个insert_log目录，占用空间好大可以删除吗？

或者有什么地方配置不产生insert_log吗，我没有在milvus.yml里面找到相应配置的地方。
![image](https://user-images.githubusercontent.com/6022635/209461715-21a5f132-fd0a-4f7d-bdfe-4f2715c60694.png)

**Comment by xiaofan-luan** [comment, marked as answer]
Insert 就是数据本身，是不能删除的。
看大小insert和index的大小很相似，为什么认为很大呢。本身一条向量就是好几k

**Reply by luokuncool** [reply]
哦，好的，我还以为是没有用的log数据😂
